Wiley Online Library.js

```diff
@@ -474,7 +475,7 @@ function scrapeCochraneTrial(doc, url){
 
 	processSubtitles(doc, item);
 	addBookReviewTag(doc, item);
-	validatePageCount(item);
+	//validatePageCount(item);
 
 	item.complete();
 }
```

Comment:
Where have the missing author problems mentioned in https://github.com/ubtue/DatenProbleme/issues/1517) been solved? missing 100 fields will generally always make our QS fail.

Reply:
Error no longer appears